bsp/nrf5x/nrf52832/board/Kconfig

```diff
@@ -313,6 +313,27 @@ endif
         int
         default 1
     endif
+
```

Comment:
Please sync the nrf52840 Kconfig to this config. by the way , add the config to the nrf52840 RTC driver